atd-etl/cr3_download/cr3_download.py

```diff
@@ -15,11 +15,6 @@
 from process.config import ATD_ETL_CONFIG
 from process.helpers_cr3 import *
 
-def wait(int):
```

Comment:
This method was not used in this script. It is removed in `helpers_cr3.py` for the same reason.